README.md

```diff
@@ -12,6 +12,8 @@ See also [Deprecated API docs](https://github.com/kefirjs/kefir/blob/master/depr
 
 
 
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/kefirjs/kefir/blob/master/LICENSE.txt)
+[![npm version](https://img.shields.io/npm/v/kefir.svg?style=flat)](https://www.npmjs.com/package/kefir)
 [![Build Status](https://travis-ci.org/kefirjs/kefir.svg?branch=master)](https://travis-ci.org/kefirjs/kefir)
 [![Dependency Status](https://david-dm.org/kefirjs/kefir.svg)](https://david-dm.org/kefirjs/kefir)
 [![devDependency Status](https://david-dm.org/kefirjs/kefir/dev-status.svg)](https://david-dm.org/kefirjs/kefir#info=devDependencies)
```

Comment:
Adding these badges is a lot of badges, and I'm wondering if maybe we should kill these dependency badges?

Reply:
I'm down for dropping both the dependency badges, or at least the devDeps badge. Seems like noise to have both together.